forked-daapd.conf

```diff
@@ -1,57 +1,256 @@
 
+# A quick guide to configuring forked-daapd:
+#
+# For regular use, the most important setting to configure is "directories",
+# which should be the location of your media. Whatever user you have set as
+# "uid" must have read access to this location. If the location is a network
+# mount, please see the README.
+#
+# In all likelihood, that's all you need to do!
+
 general {
 	# Username
+	# Make sure the user has read access to the library directories you set
+ 	# below, and full access to the databases, log and local audio
 	uid = "daapd"
-	logfile = "/log/forked-daapd/forked-daapd.log"
+
 	# Database location
-	db_path = "/cache/forked-daapd/songs3.db"
+#	db_path = "/var/cache/forked-daapd/songs3.db"
+
+	# Log file and level
 	# Available levels: fatal, log, warning, info, debug, spam
+	logfile = "/var/log/forked-daapd.log"
 	loglevel = log
+
 	# Admin password for the non-existent web interface
 	admin_password = "unused"
+
 	# Enable/disable IPv6
 	ipv6 = yes
+
+	# Location of cache database
+#	cache_path = "/var/cache/forked-daapd/cache.db"
+
+	# DAAP requests that take longer than this threshold (in msec) get their
+	# replies cached for next time. Set to 0 to disable caching.
+#	cache_daap_threshold = 1000
 }
 
 # Library configuration
 library {
 	# Name of the library as displayed by the clients
 	# %h: hostname, %v: version
-	name = "Music Server"
+	name = "My Music on %h"
+
 	# TCP port to listen on. Default port is 3689 (daap)
 	port = 3689
+
 	# Password for the library. Optional.
 #	password = ""
 
 	# Directories to index
-	directories = { "/media" }
-	# Directories containing compilations
-	# Matches anywhere in the path (not a regexp, though)
-#	compilations = { "/compilations/" }
+	directories = { "/srv/music" }
```

Comment:
This and the other media directories should be in the /media volume. If all are under /media then the media volume can be moved using an environment variable, see README.md and start.sh.